resp2_replies.json

```diff
@@ -461,6 +461,23 @@
   "FLUSHDB": [
     "[Simple string reply](../topics/protocol.md#simple-strings): `OK`."
   ],
+  "FT.CREATE": [
+    "[Simple string reply](../topics/protocol.md#simple-strings): `OK`.",
+    "",
+    "If an error occurs during the index creation, an error is returned."
```

Comment:
I don't think we explicitly call out error responses. I don't see another example anywhere.